src/dataflow-types/logging.rs

```diff
@@ -525,7 +525,7 @@ const VIEW_PERF_ARRANGEMENT_RECORDS: LogView = LogView {
     name: "mz_perf_arrangement_records",
     sql: "CREATE MATERIALIZED VIEW mz_perf_arrangement_records AS SELECT mas.worker, name, records, operator
 FROM mz_catalog.mz_arrangement_sizes mas
-JOIN mz_catalog.mz_dataflow_operators mdo ON mdo.id = mas.operator",
+LEFT JOIN mz_catalog.mz_dataflow_operators mdo ON mdo.id = mas.operator",
```

Comment:
this is not your fault at all but I also think this view should be joining on worker - because operators on different threads share the same id and name

```
materialize=> select * from mz_dataflow_operators where id = 512;
 id  | worker |     name
-----+--------+--------------
 512 |      0 | JoinStage: 0
 512 |      1 | JoinStage: 0
 512 |      2 | JoinStage: 0
 512 |      3 | JoinStage: 0
(4 rows)

materialize=> select * from mz_arrangement_sizes where operator = 512;
 operator | worker | records | batches
----------+--------+---------+---------
      512 |      0 |     100 |       2
      512 |      1 |      89 |       2
      512 |      2 |     103 |       2
      512 |      3 |      88 |       2
(4 rows)

materialize=> select * from mz_perf_arrangement_records where operator = 512;
 worker |     name     | records | operator
--------+--------------+---------+----------
      0 | JoinStage: 0 |     100 |      512
      0 | JoinStage: 0 |     100 |      512
      0 | JoinStage: 0 |     100 |      512
      0 | JoinStage: 0 |     100 |      512
      1 | JoinStage: 0 |      89 |      512
      1 | JoinStage: 0 |      89 |      512
      1 | JoinStage: 0 |      89 |      512
      1 | JoinStage: 0 |      89 |      512
      2 | JoinStage: 0 |     103 |      512
      2 | JoinStage: 0 |     103 |      512
      2 | JoinStage: 0 |     103 |      512
      2 | JoinStage: 0 |     103 |      512
      3 | JoinStage: 0 |      88 |      512
      3 | JoinStage: 0 |      88 |      512
      3 | JoinStage: 0 |      88 |      512
      3 | JoinStage: 0 |      88 |      512
(16 rows)

materialize=>
```